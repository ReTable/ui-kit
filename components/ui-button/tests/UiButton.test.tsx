import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import { UiButton } from '~';

describe('UiButton', () => {
  it('works', () => {
    const tree = create(<UiButton />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
