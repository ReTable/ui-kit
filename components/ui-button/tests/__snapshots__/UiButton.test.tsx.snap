// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiButton > works 1`] = `
<div
  className="UiButton_root__1w7vco90 "
/>
`;
